app/javascript/queries/github_user_queries.js

```diff
@@ -0,0 +1,15 @@
+import gql from "graphql-tag";
+
+const GITHUB_USERS_QUERY = gql`
+  query GITHUB_USERS_QUERY {
```

Comment:
can we camel case these queries? Mostly what I've seen online and what I've always done

Reply:
Completed in https://github.com/oddballio/magnifier/pull/60/commits/034d068ef1c5b1c472ace3156c98fd9e9960c542